app/js/datepicker.jsx

```diff
@@ -189,6 +197,58 @@ class DatePicker extends React.Component {
 
     }
 
+    handleStartDateInputChange(e) {
+        this.setState({
+            startDateInputValue: e.target.value
+        });
+    }
+
+    handleEndDateInputChange(e) {
+        this.setState({
+            endDateInputValue: e.target.value
+        });
+    }
+
+    handleStartDateKeyPress(e) {
+        if (e.charCode === CHAR_CODE_ENTER) {
```

Comment:
this is a deprecated event field: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/charCode